app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -103,7 +103,7 @@ class NewMessageFragment : Fragment() {
     private var quoteWebView: WebView? = null
     private var signatureWebView: WebView? = null
 
-    private val signatureAdapter = SignatureAdapter(::onSignatureClicked)
+    private var signatureAdapter: SignatureAdapter? = SignatureAdapter(::onSignatureClicked)
```

Comment:
If the value is made null inside onDestroyView maybe we want to instantiate it inside onCreateView